muikku-core-plugins/src/main/resources/META-INF/resources/scripts/gui/workspace-materials.js

```diff
@@ -85,7 +85,8 @@
   $(window).load(function () {
     // Workspace's material's TOC
     if ($('#workspaceMaterialsTOCWrapper').length > 0) {
-      
+
+      var contentContainer = ($('#contentWorkspaceMaterials').length > 0 ? contentContainer = $('#contentWorkspaceMaterials') : contentContainer = $('#content'));
```

Comment:
These assignment expressions (the two latter "contentContainer ="'s) are redundant and confusing.